parquet_file/src/catalog.rs

```diff
@@ -1103,7 +1103,7 @@ impl<'c> TransactionHandle<'c> {
     /// Remove a parquet file from the catalog.
     ///
     /// Removing files that do not exist or were already removed will result in an error.
```

Comment:
```suggestion
    /// Remove a parquet file from the catalog.
```

I think the comment needs to be updated